test/scripts/init.js

```diff
@@ -31,41 +32,58 @@ describe('init', () => {
     });
   }
 
-  function compareFile(a, b) {
+  async function compareFile(a, b) {
     const streamA = createSha1Hash();
     const streamB = createSha1Hash();
 
-    return Promise.all([
-      pipeStream(fs.createReadStream(a), streamA),
-      pipeStream(fs.createReadStream(b), streamB)
-    ]).then(() => streamA.read().equals(streamB.read()));
+    await Promise.all([
+      pipeStream(createReadStream(a), streamA),
+      pipeStream(createReadStream(b), streamB)
+    ]);
+
+    return streamA.read().equals(streamB.read());
   }
 
-  function check(path) {
-    return Promise.each(assets, item => compareFile(
-      pathFn.join(assetDir, item),
-      pathFn.join(path, item)
-    ).should.eventually.be.true).finally(() => rmdir(path));
+  async function check(path) {
+    for (const item of assets) {
+      const result = await compareFile(
+        join(assetDir, item),
+        join(path, item)
+      );
+
+      result.should.be.true;
```

Comment:
It seems this line is never executed because the assets always empty.

Reply:
I can confirm, I think CI doesn't initialize the `assets` submodule.